README.md

```diff
@@ -207,6 +207,10 @@ This will only be used if the size cannot be automatically calculated. This
 is currently only used and required if you supply a `Readable` stream as the
 file to upload. You may also use this to limit the position until which a file
 will be uploaded.
+* `agent = undefined`: specifies the [Agent](https://nodejs.org/api/http.html#http_class_http_agent)
```

Comment:
We usually use `null` as a default value for optional parameters. Please use it and also update the code.